lib/yrewrite/seo.php

```diff
@@ -245,13 +254,21 @@ public function sendSitemap($domain = '')
                             }
                         }
 
-                        $sitemap[] =
+                        $sitemap_entry =
                           "\n".'<url>'.
-                          "\n".'<loc>'.rex_yrewrite::getFullPath($path[$clang_id]).'</loc>'.
-                          "\n".'<lastmod>'.date(DATE_W3C, $article->getUpdateDate()).'</lastmod>'. // serverzeitzone passt
-                          "\n".'<changefreq>'.$changefreq.'</changefreq>'.
-                          "\n".'<priority>'.$priority.'</priority>'.
+                          "\n\t".'<loc>'.rex_yrewrite::getFullPath($path[$clang_id]).'</loc>'.
+                          "\n\t".'<lastmod>'.date(DATE_W3C, $article->getUpdateDate()).'</lastmod>'; // serverzeitzone passt
+                        if($article->getValue(self::$meta_image_field)) {
+                            $media = rex_media::get($article->getValue(self::$meta_image_field));
+                            $sitemap_entry .= "\n\t".'<image:image>'.
+                                "\n\t\t".'<image:loc>'.rtrim(rex_yrewrite::getDomainByArticleId($article->getId())->getUrl(), "/").$media->getUrl().'</image:loc>'.
+                                ($media->getTitle() ? "\n\t\t".'<image:title>'.rex_escape(strip_tags($media->getTitle())).'</image:title>' : '').
+                                "\n\t".'</image:image>';
```

Comment:
Hier dann wie besprochen das Bild über den Media-Manager referenzieren.
Wobei noch die Frage ist, welchen MM-Typ wie nun nehmen. Einen der schon existierenden Standard-Typen, oder einen spezifischen, wie `yrewrite_seo_image` oder so. Letzteren könnte man dann optional bei sich anlegen/konfigurieren, so wie es bereits mit `yrewrite_default` ist.

Bisschen offen ist aber auch noch meine Frage [von weiter oben])https://github.com/yakamara/redaxo_yrewrite/pull/461/files#r841049773).

Reply:
Vorschlag: wir verwenden den Typ: rex_media_medium. Der sollte für Google passen.

Reply:
Wichtig ist hier vor allem das erste Bild. Für weitere Bilder sind mir bisher eine Anwendungsszenarien bekannt. Es wäre ein leichtes dieses Feld später zu erweitern, falls es nötig wird. Daher würde ich es derzeit bei einem Bild belassen.

Reply:
Die Abhängigkeit in der package.yml ist hinzugefügt.

Reply:
Ich markiere es mal als resolved. Jetzt haben wir eine Basic-Lösung, die sich jederzeit um Features erweitern lässt, wenn Bedarf entsteht.